drivers/platform/chibios/chibios.mk

```diff
@@ -0,0 +1,29 @@
+#linking no-os driver 
+
+#no-os headers
+NOOS_INC	:=	$(NOOS)/include/
```

Comment:
the indentation is this file still is not solved. you can take as examples some other makefiles used in the no-os projects.:
https://github.com/analogdevicesinc/no-OS/blob/master/projects/eval-ade9430/src.mk